### ISSUE TITLE
Redaction Failing

### ISSUE THREAD

**Issue by CorrinDuque** [zendesklabs/ticket_redaction_app#49]
We are running into functionality issues.  All process steps seem to function fine when using this app but then the result is an error message of "some of your redactions failed" where ALL redactions fail.  We decided to attempt to redact only the acronym 4WS which also repeatedly failed.  I worked with an agent today where we both made more than 10 attempts to redact a short portion of script on a ticket and the redaction failed.  After about 20 attempts (cumulative) the redaction of 4WS processed successfully.

We are on version 1.7 that states it was updated 22 days ago.  Is there a new release of this app?

**Comment by dpawluk** [comment]
@CorrinDuque Thanks for the report. Currently there are some improvements that need to be made because of how ticket updates are handled, meaning one comment can be redacted at a time. Also originally the code was written before html was used in tickets (plain text only) and so that is another enhancement that will have to be made. Since this is free open-source software it will probably be a bit before time is found to spend refactoring the code to work with rich text and queue requests. Ideally this can be done using the framework v2.

TLDR; realistically this will probably take between 1-2 months to get done right. Sorry for any inconvenience. Any additional data I can use (regarding your settings for agent text editing/rich vs plain vs markdown/ etc, will certainly help make the effort go faster. This is open source software so if you do have any engineering resources pull requests are very welcome. Thanks for participating in Zendesk Labs!

**Comment by CorrinDuque** [comment]
Thanks Daniel!  Much appreciated.  Since Zendesk just announced today that they will be removing support for plain text (and they have made improvements to the rich text editor) we are now using the rich text editor.  We made that switch to rich text yesterday.  The redaction issue was today.  Although we do not have a ton of redaction opportunities/needs I did not receive any agent reports of issues prior to our switch over to rich text today. 

Could be a cache issue so I will clear history and do some testing via incognito.

**Comment by CorrinDuque** [comment]
Hi Daniel,
I did a bit of testing via incognito and did not run into any failure pop ups or functionality issues.  This might have been a cache issue as we just switched over to Rich Text yesterday due to added feature capabilities with added to the rich text editor.  I'll let you know if we run into issues again.
